modules/nw-egress-router-about.adoc

```diff
@@ -99,6 +99,14 @@ $ openstack port set --allowed-address \
   ip_address=<ip_address>,mac_address=<mac_address> <neutron_port_uuid>
 ----
 
+[NOTE]
+====
+The {rh-openstack} egress IP address feature creates a neutron reservation port called `egressip-<IP address>`.
+When an egress IP address on an OpenStack network is moved from one node to another, because of a node failover for example, the neutron reservation port is removed and recreated. As a result, a floating IP address might be bound to the port, creating a SNAT mapping of the egress IP address to a public IP address. Because SNAT breaks the floating IP address, a system administrator must recreate the floating IP address.
```

Comment:
This sounds inaccurate to me. First of all we should probably say about assigning floating IPs to the reservation port before we mention the failover issue. First part if the issue is described correctly, second is not, it's not SNAT breaking the FIP, it's just fact of port recreation. And we don't need to recreate the FIP, we just need to attach it again to the new reservation port. It shouldn't need admin credentials either.

My draft:
> You can assign a floating IP address to such a reservation port in order to have a predictable SNAT address for egress traffic going outside of the cluster. Note that currently when an egress IP address on an OpenStack network is moved from one node to another, because of a node failover for example, the neutron reservation port is removed and recreated. This means that floating IP association will be lost and you need to manually reassign the FIP to the new reservation port.

It might be cool to say that this limitation is planned to be resolved in future versions.

Reply:
Thanks @dulek ...I made the update with a couple of style tweaks. Also, in the IBM Style Guide, it states: 
"Future product releases: Avoid claims that are related to the content or timing of a future product or release." When this changes, I can update the content accordingly. 

Also, I want to change the first sentence to this: (Remove 'such')

"You can assign a floating IP address to a reservation port in order to have a predictable SNAT address for egress traffic going outside of the cluster." 

Your thoughts?
Thanks!

Reply:
Sure, it works. But you've also removed the "The {rh-openstack} egress IP address feature creates a neutron reservation port called `egressip-<IP address>`." part that I expected to be left as is. And that is why I had "such" in the next sentence.